src/python/pants/bin/goal_runner.py

```diff
@@ -103,22 +88,9 @@ def _init_graph(self,
     if not graph_helper:
       native = Native.create(self._global_options)
       native.set_panic_handler()
-      graph_helper = EngineInitializer.setup_legacy_graph(
-        pants_ignore_patterns,
-        workdir,
-        self._global_options.build_file_imports,
-        glob_match_error_behavior=GlobMatchErrorBehavior.create(
-          self._global_options.glob_expansion_failure),
-        native=native,
-        build_file_aliases=self._build_config.registered_aliases(),
-        rules=self._build_config.rules(),
-        build_ignore_patterns=build_ignore_patterns,
-        exclude_target_regexps=exclude_target_regexps,
-        subproject_roots=subproject_build_roots,
-        include_trace_on_error=self._options.for_global_scope().print_exception_stacktrace,
-        remote_store_server=self._options.for_global_scope().remote_store_server,
-        remote_execution_server=self._options.for_global_scope().remote_execution_server,
-      ).new_session()
+      graph_scheduler_helper = EngineInitializer.setup_legacy_graph(native,
+                                                                    self._global_options)
```

Comment:
seems like several kwargs that were explicitly passed in before from options at this callsite (e.g. `glob_match_error_behavior`, `rules`, `include_trace_on_error`, `build_file_aliases` etc) are now omitted in the new call to `setup_legacy_graph` - is that expected?

Reply:
Good eye! Hence the CI failures. Thanks.